tree/doc/v528/index.html

```diff
@@ -89,7 +89,7 @@ <h3>Tree</h3>
 *Entries :       20 : Total  Size=       1398 bytes  File Size  =        400 *
 *Baskets :        1 : Basket Size=      16000 bytes  Compression=   2.23     *
 *............................................................................*
-*Br   19 :fIsValid  : Bool_t                                                 *
```

Comment:
Was this intended? This is the doc of ROOT 5.

Reply:
Thanks, I missed that this part of the doc was for ROOT 5. I'll remove this change from the commit.

Reply:
In addition this output (i.e. `TTree::Print`) is not changed by this PR.